opt/Makefile

```diff
@@ -156,7 +156,7 @@ clean:
 ifeq ($(ALL),1)
 	$(SHOW)if [ -d "$(BINROOT)" ]; then rm -rf $(BINROOT); fi
 	$(SHOW)if [ -d "$(INSTALL_DIR)" ]; then rm -rf $(INSTALL_DIR); fi
-	$(SHOW)rm -f $(ROOT)/install-$(DEVICE)
+	$(SHOW)rm -rf $(ROOT)/install-$(DEVICE)
```

Comment:
This (e.g. `install-gpu` or `install-cpu`) is supposed to be a symlink. Not recursing would suffice in that case. Did you incur in a situation where recursion was necessary?

Reply:
Must have been an artifact from an earlier build. Started clean and I didn't need to recurse. Corrected PR.